examples/index.json

```diff
@@ -570,7 +570,7 @@
                 "name": "Sequence Diagrams",
                 "url": "sequence-diagrams",
                 "verifyBuild": true,
-                "verifyOutput": true
+                "verifyOutput": false
```

Comment:
Non intensional?

Reply:
It was intentional. But seems like the `mocky.io` fails temporarily. Retriggered to check again.